packages/client/lib/MetadataClient.ts

```diff
@@ -10,7 +7,10 @@ import {
   OpenID4VCIServerMetadata,
   OpenIDResponse,
   WellKnownEndpoints,
-} from './types';
+} from '@sphereon/openid4vci-common/lib';
```

Comment:
This is referencing a dir in the package, suggesting the exports are not in order. Should not have to include /lib

Reply:
exports were fine. Intellij auto-refactor upone move added the folder.
Removed the lib part from path.